OracleTuxedo/OracleTSAM/dockerfiles/12.2.2/Dockerfile

```diff
@@ -0,0 +1,130 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1996-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) Oracle Tuxedo System and Applications Monitor (TSAM) Plus 12cR2 (12.2.2) GA Installer: tsam122200_64_Linux_x86.zip
+#     Download from http://www.oracle.com/technetwork/middleware/tuxedo/downloads/index.html
+#
+# (2) Oracle TSAM Plus 12.2.2 Rolling Patch 003: p25448043_12220_Linux-x86-64.zip
+#     Download from https://updates.oracle.com/Orion/Services/download/p25448043_12220_Linux-x86-64.zip?aru=20968945&patch_file=p25448043_12220_Linux-x86-64.zip
+#
+# (3) Oracle Database Instant Client: instantclient-basiclite-linux.x64-12.2.0.1.0.zip & instantclient-sqlplus-linux.x64-12.2.0.1.0.zip
+#     Download from http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run:
+#      $ docker build -t oracle/tsam:12.2.2.1 .
+
+# Pull base image
+# ---------------
+FROM oracle/serverjre:8
+
+# Maintainer
+# ----------
+MAINTAINER Chris Guo <chris.guo@oracle.com>
+
+# Common environment variables required for this build (do NOT change)
+# --------------------------------------------------------------------
+ENV ORACLE_HOME=/u01/oracle \
+    PATH=$PATH:/usr/java/default/bin \
+    TMPFILES=/tmp/files
+
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
+# ------------------------------------------------------------
+RUN mkdir -p /u01/oracle && \
+    chmod a+xr /u01 && \
+    useradd -u 1000 -b /u01 -M -s /bin/bash oracle && \
+    chown oracle:oracle -R /u01 && \
+    yum install -y libaio vim tar net-tools openssh-server rsync sudo && \
```

Comment:
Why are you installing OpenSSH, rsync and sudo? These should not be necessary inside a Docker container.